tests/README.md

```diff
@@ -43,6 +43,8 @@ Any time changes made to the operator are being checked into the main branch, sa
   - (if running sanity, powerflex, or modules suites) `test-vxflexos`
   - (if running sanity, powerscale, or modules suites) `isilon`
   - (if running unity suite) `unity`
+  - (if running powermax e2e tests) `powermax`
```

Comment:
can we match the style of the above bullets -- e.g., (if running powermax suite) not (if running powermax e2e tests).

Reply:
I have added these changes now. Thank you for the review!